lib/ansible/modules/database/mongodb/mongodb_user.py

```diff
@@ -25,69 +25,85 @@
 options:
     login_user:
         description:
-            - The username used to authenticate with
+            - The MongoDB username used to authenticate with.
+        type: str
     login_password:
         description:
-            - The password used to authenticate with
+            - The login user's password used to authenticate with.
+        type: str
     login_host:
         description:
-            - The host running the database
+            - The host running the database.
         default: localhost
+        type: str
     login_port:
         description:
-            - The port to connect to
-        default: 27017
+            - The MongoDB port to connect to.
+        default: '27017'
+        type: str
     login_database:
         version_added: "2.0"
         description:
-            - The database where login credentials are stored
+            - The database where login credentials are stored.
+        type: str
     replica_set:
         version_added: "1.6"
         description:
-            - Replica set to connect to (automatically connects to primary for writes)
+            - Replica set to connect to (automatically connects to primary for writes).
+        type: str
     database:
         description:
-            - The name of the database to add/remove the user from
+            - The name of the database to add/remove the user from.
         required: true
+        type: str
+        aliases: [db]
     name:
         description:
-            - The name of the user to add or remove
+            - The name of the user to add or remove.
         required: true
-        aliases: [ 'user' ]
+        aliases: [user]
+        type: str
     password:
         description:
-            - The password to use for the user
+            - The password to use for the user.
+        type: str
+        aliases: [pass]
     ssl:
         version_added: "1.8"
         description:
-            - Whether to use an SSL connection when connecting to the database
+            - Whether to use an SSL connection when connecting to the database.
         type: bool
     ssl_cert_reqs:
         version_added: "2.2"
         description:
             - Specifies whether a certificate is required from the other side of the connection, and whether it will be validated if provided.
-        default: "CERT_REQUIRED"
-        choices: ["CERT_REQUIRED", "CERT_OPTIONAL", "CERT_NONE"]
+        default: CERT_REQUIRED
+        choices: [CERT_NONE, CERT_OPTIONAL, CERT_REQUIRED]
+        type: str
     roles:
         version_added: "1.3"
+        type: list
```

Comment:
```suggestion
        type: list
        elements: str
```
i think it allows us to remove one more line from ignore.txt